DBInsertData.sql

```diff
@@ -8,14 +8,14 @@ Insert INTO Class VALUES ('JV123', 'FSD123', 'IN5');
 
 Insert INTO Employees VALUES('IM505', 'IceMan', 'IM505@syn.com', 'JV123');
 
-Insert INTO Type VALUES('Java', 'FSD123');
-Insert INTO Type VALUES('DB', 'DB343');
+--Insert INTO Type VALUES('Java', 'FSD123');

```

Comment:
These don't need to be commented out, because the script is going to be run by someone who has never run it before.